var/spack/repos/builtin/packages/fastdfs/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fastdfs(Package):
```

Comment:
This file is in the right directory but the install method needs to be updated.

Reply:
I fixed it in cfdacd1